lib/api/globals/index.ts

```diff
@@ -0,0 +1,85 @@
+import { gql } from "graphql-request";
+import { fallbackLng } from "@/lib/i18n/settings";
+import { queryAPI } from "@/lib/fetch";
+import { linkFragment } from "@/lib/api/fragments/link";
+import {
+  siteInfoFragment,
+  footerFragment,
+  rootPageInfoFragment,
+  contactFormFragment,
+} from "@/lib/api/fragments/global";
+import { categoriesFragment } from "@/lib/api/fragments/categories";
+import { userProfileFragment } from "@/lib/api/fragments/page";
+import { getSiteFromLocale } from "@/lib/helpers/site";
+
+export async function getGlobalData(locale = fallbackLng) {
```

Comment:
moved out to a new file, app directory porting will require moving some stuff around since a lot of API calls are done in files that are mixed with `useSWR` which is client-only.